### PR TITLE
Update 3dconnexion from 10-7-0 to 10-7-2

### PR DESCRIPTION
**Pull request by MTCoster**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

---

Apologies for the bad CI run - `style --fix` broke things by putting `url` before `version`

### CODE REVIEW

File: Casks/3dconnexion.rb [hunk 1]
```diff
@@ -1,39 +1,31 @@
...
-
-    livecheck do
-      url "https://3dconnexion.com/us/support/faq/beta-driver-for-macos-11-big-sur/"
-      regex(/href=.*?3DxWareMac_v(\d+(?:-\d+)*_r\d+)\.dmg/i)
-    end
+    version "10-7-2,r3454,A2E276BD-4DEB-4343-BA4C-591FE7958ED9"
+    sha256 "a53258a8b89d511182dd5427140ba7d1c539a152d87b255f1553704bab7413ad"
   end
 
+  url "https://download.3dconnexion.com/drivers/mac/#{version.csv[0]}_#{version.csv[2]}/3DxWareMac_v#{version.csv[0]}_#{version.csv[1]}.dmg"
```
Comment (bevanjkay) [inline Casks/3dconnexion.rb:10]: @MTCoster Generally the trend is to use `version.csv.first` / `version.csv.second` / `version.csv.third`

File: Casks/3dconnexion.rb [hunk 2]
```diff
@@ -1,39 +1,31 @@
...
+    end
+  end
+
   depends_on macos: ">= :yosemite"
 
   pkg "Install 3Dconnexion software.pkg"
 
   uninstall pkgutil:   "com.3dconnexion.*",
-            launchctl: "com.3dconnexion.nlserverIPalias",
+            launchctl: "com.3dconnexion.*",
```
Comment (bevanjkay) [inline Casks/3dconnexion.rb:28]: @MTCoster I don't believe that globs are functional in launchctl, so they need to be individually listed

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/3dconnexion.rb (hunk 2 above)
Comment (MTCoster) [reply Casks/3dconnexion.rb]: Wasn't sure about that one, thanks!

File: Casks/3dconnexion.rb (hunk 1 above)
Comment (MTCoster) [reply Casks/3dconnexion.rb]: Fixed - these just came out of the old version. Is it worth also replacing the first part of the version string with `10.7.2` so it's more accurate and using a `gsub` on it get `10-7-2`?

**Comment by MTCoster** [comment]
Any idea why the install is timing out on the macos-12 build agent? It works fine for me locally running Monterey 12.3.1

**Pull request merged**